packages/lib/universal-sql/src/client-duckdb/node.js

```diff
@@ -67,6 +67,11 @@ export async function initDB() {
 			}
 		});
 		connection = db.connect();
+
+		// revert breaking changes
```

Comment:
Please elaborate here about what the breaking changes were, between what versions, and how this fixes it